Source/Main.cpp

```diff
@@ -149,9 +152,26 @@ int main(int argc, char *argv[])
                 MainMenu->normal_quit_button_instance->render();
             }
         }
-
-        frontHopper.render();
-        backHopper.render();
+        // Same block of if's as above but for pause screen
```

Comment:
This I'm not a massive fan of, it's repeated code, which sorta defeats the purpose of an object.